recipes/vs2015_runtime/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: vs2015_runtime
+  # version is the update version reported from the registry
+  version: "14.0.25123"
+
+build:
+  number: 0
+  msvc_compiler: 14.0
+  skip: True  # [not win]
+
+test:
+  commands:
+    # only test one of the api dll's.  We test all the others, but there are lots of these.
+    - if not exist Library\bin\api-ms-win-crt-string-l1-1-0.dll exit 1
```

Comment:
Indeed. I try to not keep too much of the windows details in my mind, so it took me another shot. Also, I failed to use `%LIBRARY_BIN%`. But we'll see how it goes...